common/library/module_utils/input_validation/common_utils/validation_utils.py

```diff
@@ -570,6 +570,25 @@ def check_bmc_static_range_overlap(static_range, static_range_group_mapping) ->
 
     return grp_overlaps
 
+def get_interface_ip_and_netmask(interface):
+    """
+    Returns the IPv4 address and netmask bits assigned to an interface, or (None, None) if not found.
+    """
+    try:
+        result = subprocess.run(
```

Comment:
can we verify this logic in lom case where more than 2 or 3 ip present, use inbuilt function if available